manifest.json

```diff
@@ -1,21 +1,26 @@
 {
+    "manifest_version": 2,
     "name": "Totally for real actual ghostery extension",
     "version":"0.000001",
-    "manifest_version": 2,
     "content_scripts": [
         {
             "matches": ["<all_urls>"],
-            "js": ["content.js"]
+            "js": ["content-scripts/friendly-notification.js"]
         }
     ],
+
     "browser_action": {
         "default_icon": {
-            "16": "icon99.png",
-            "24": "icon99.png",
-            "32": "icon99.png"
+            "16": "images/icon99.png",
+            "24": "images/icon99.png",
+            "32": "images/icon99.png"
```

Comment:
It's standard practice for the number in the icon filename to signify its pixel dimensions. You can still keep the `99` if you like - I would then just do something like `icon99_32`, etc.

Also, keep in mind that different keys in the `default_icon` object should have values that are icons with matching dimensions (so for example the value for the `16` key should be a 16x16 icon). The reason is that the different icons are used in different places in the Chrome interface; more details at https://developer.chrome.com/apps/manifest/icons